### DISCUSSION TITLE
Publish of Developer Portal fails - no logs

### DISCUSSION THREAD

**Discussion by remonboonstra** [Azure/api-management-developer-portal#2617] [Feature request]
## Bug description

I've only used the UI to customize the Developer Portal and it is now not possible to Publish anymore.
As there are no logs about the publishing process, it is a black hole? I can't figure out how to solve it.

Even reverting to a previous version is not helping anymore.

Also tried deploying to second instance using (scripts.v2) but same happens there.

## Reproduction steps

Go to Azure APIM Management blade.
Navigate to Portal Overview > Revisions
See the Status show Failed.
Nothing more can be found here about why it failed.

## Expected behavior

**Feature request:** There should really be a possibility to see a log on the publishing process OR at least the error (+details) on why it failed to publish. That way users might be able to fix the issue themselves. 

## Is your portal managed or self-hosted?

Managed

## API Management service name

gmp-dev-apim

## Additional context

![image](https://github.com/Azure/api-management-developer-portal/assets/56164388/6a9589f4-01d9-4871-8b77-81249b2e3c2a)

**Comment by sthirthala** [comment]
For managed portal, please log support tickets in Azure help+support center

https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request

**Comment by remonboonstra** [comment]
@sthirthala please don’t close this with this response. This is not an answer, this is exactly what I don’t want to achieve! We don’t want to be left in the dark when a failure occurs. Please fix or put on the backlog; instead of closing with a wrong answer. Thank you

**Comment by sthirthala** [comment]
@remonboonstra  can you please raise a support request so our team can investigate it.

**Comment by sthirthala** [comment]
We need correlation id and other information to investigate. Hence creating support ticket for investigation would be best way to go. The Github issues list was meant for bug reports only. I am fine leaving this issue open - but best to create support ticket to start investigation.

**Comment by remonboonstra** [comment]
@sthirthala thank you! I already went through the support, and we fixed it by starting from scratch !? 
This is **not** a request for support, this is a request for **making the product better**. Thanks!

**Comment by mrcarlosdev** [comment]
This issue is related to managed developer portal. We advise you to create a Azure support request to get assistance on this issue. Please refer to the below link to create a new Azure support request, Please select Problem Type = "Developer Portal" in the request to route it appropriately.

https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request
